package.json

```diff
@@ -31,9 +30,12 @@
     "uglify-js": "^2.4.16",
     "watchify": "^2.2.1"
   },
+  "peerDependencies": {
+    "coffeeify": "^1.0.0"
```

Comment:
I haven't been able to pick up coffeeify as a dependency on a fresh `npm i`, is yours locating it okay?